packages/styling/src/styles/theme.ts

```diff
@@ -18,17 +18,21 @@ let _onThemeChangeCallbacks: Array<(theme: ITheme) => void> = [];
 
 export const ThemeSettingName = 'theme';
 
-if (!Customizations.getSettings([ThemeSettingName]).theme) {
-  const win: any = getWindow(); // tslint:disable-line:no-any
+export function initializeThemeInCustomizations(): void {
```

Comment:
oh one more minor change; if there is any content in this file which could be stripped in tree shaking, it should be in its own file. I'm not sure there is, so don't feel compelled to change anything but just a mention to look for global code and isolate (perhaps we should follow the filename convention `*.global.ts*` to separate side effect code from treeshakeable.)

Reason: Webpack has unpredictable results when it finds a module with side effect code and ends up pulling in more than it needs. Shift/Marija found this out a few weeks ago.

Reply:
I'm no longer marking `theme.ts` as a `sideEffect`. therefore there is no need to split this file further